base/strings/string.jl

```diff
@@ -85,7 +85,7 @@ function unsafe_takestring(m::Memory{UInt8})
 end
 
 """
-    unsafe_string(p::Ptr{UInt8}, [length::Integer])
+    unsafe_string(p::Union{Ptr{UInt8},Cstring}, [length::Integer])
```

Comment:
Hmm, the `Cstring` versions is unary. So perhaps
```suggestion
    unsafe_string(p::Ptr{UInt8}, [length::Integer])
    unsafe_string(p::Cstring)
```
But I am not completely sure if this is supposed to be "official". If it is, then one can also wonder about the methods dealing with "wide" strings: `Cwstring` and `Ptr{Union{Int32, UInt16, UInt32}}`